apps/oauth2/lib/Controller/SettingsController.php

```diff
@@ -81,7 +81,7 @@ public function __construct(string $appName,
 
 	public function addClient(string $name,
 							  string $redirectUri): JSONResponse {
-		if (filter_var($redirectUri, FILTER_VALIDATE_URL) === false) {
+		if (!\OCP\Util::isValidUrl($redirectUri)) {
```

Comment:
Sorry, small detail, `OCP\Util` could be imported: `use OCP\Util;`
```suggestion
		if (!Util::isValidUrl($redirectUri)) {
```
like it's done in `apps/theming/lib/Controller/ThemingController.php`